pkg/auth/providers/common/provider.go

```diff
@@ -12,6 +12,11 @@ const (
 	UserAttributePrincipalID = "principalid"
 	// UserAttributeUserName is a key in the ExtraByProvider field of the UserAttribute that holds usernames for a partucular provider.
 	UserAttributeUserName = "username"
+	// ExtraRequestTokenID is the key for the request token ID in the UserInfo's extra attributes.
+	ExtraRequestTokenID = "cattle.io/requesttokenid"
+	// ExtraRequestTokenID is the key for the request host name in the UserInfo's extra attributes.
```

Comment:
```suggestion
	// ExtraRequestHost is the key for the request host name in the UserInfo's extra attributes.
```

Reply:
Good catch, thanks! 👍 Fixed.
Not sure if something like this raises to level of blocked (changes requested) though. Comments without an approval would suffice.